cluster/gce/configure-vm.sh

```diff
@@ -639,6 +644,28 @@ function run-salt() {
   salt-call --local state.highstate || true
 }
 
+function wait-for-cluster-healthy-or-bust() {
+  local attempt=0
+  local healthy_node_count=0
+  local total_node_count=0
+  echo "== waiting for cluster to be healthy =="
+  while true; do
+    healthy_node_count=$(kubectl --kubeconfig=/var/lib/kubelet/kubeconfig get nodes | grep -v master | awk '{print $3}' | grep -c Ready)
+    total_node_count=$(kubectl --kubeconfig=/var/lib/kubelet/kubeconfig --no-headers=true get nodes | grep -v master | awk '{print $3}' | wc | awk '{print $1}')
+    if [[ "${healthy_node_count}" == "${total_node_count}" ]]; then
+      break
+    fi
+    echo "== Waiting for nodes to all be healthy =="
+    if (( attempt > 5 )); then
```

Comment:
5 + 10 + 15 + 20 + 25 == 1.5 minutes (after this node is fully configured) that seems reasonable to me, but I'm happy to extend.  What number do you want?